spec/factories/obituaries.rb

```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+  factory :obituary do
+    first_name { "MyString" }
+    last_name { "MyString" }
+    age { 1 }
+    city { "MyString" }
+    state { "MyString" }
+    description { "MyString" }
+    image_url { "MyString" }
```

Comment:
Still need to add faker to these.

Reply:
@stellakunzang is really good at that :)

Reply:
yay Faker!

Reply:
Nice use here